MoltenVK/MoltenVK/GPUObjects/MVKBuffer.mm

```diff
@@ -79,11 +79,13 @@
 
 	MVKResource::bindDeviceMemory(mvkMem, memOffset);
 
-#if MVK_MACOS
```

Comment:
Should we leave the preprocessor platform checks in? This is all dead code on iOS and tvOS. Then again, I suppose running the simulator on an Intel Mac could result in needing this special handling...

Reply:
Good point. Fixed.